tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/switchcase.lg

```diff
@@ -0,0 +1,8 @@
+# greetInAWeek
+-SWITCH: {day}
+-CASE: {'Saturday'}
```

Comment:
Leave whitespace to make it more readable

Reply:
will add some whitespace to make lg file more readable